gapic/src/main/com/google/gapid/widgets/DrawerComposite.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.widgets;
+
+import static com.google.gapid.widgets.Widgets.createComposite;
+import static com.google.gapid.widgets.Widgets.createHorizontalSash;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.GC;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.graphics.Rectangle;
+import org.eclipse.swt.layout.FillLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Layout;
+import org.eclipse.swt.widgets.Sash;
+
+public class DrawerComposite extends Composite {
+  private static final int SASH_HEIGHT = 5;
+  private static final int MIN_SIZE = 50;
+  private static final int TITLE_MARGIN = 5;
+  private static final int BORDER_HEIGHT = 3;
+
+  private final int titleHeight;
+  private final Composite main;
+  private final Sash sash;
+  private final Label border;
+  private final Composite drawer;
+  private String title = "";
+  private boolean expanded = false;
+  private int drawerHeight = 150;
```

Comment:
Just a UX feedback: Would it be good if we make the initial hint for height larger? Or having the Selection view scrollable vertically would be even better.
It's just that at this height some rows are hided "perfectly" based on the font size on my machine, if I am a user for the first time and I don't see some half-covered text, I may never know there's still some fields hiding underneath.

Reply:
You're right, this is not a good value and what I just used to get going... Clearly this isn't quite ready yet. I will make it default to a better value and then remember it between runs, since the user can adjust the height.

Reply:
Done.